code/modules/food_and_drinks/restaurant/customers/_customer.dm

```diff
@@ -84,3 +84,20 @@
 		var/mutable_appearance/flag = mutable_appearance(customer.icon, "french_flag")
 		flag.appearance_flags = RESET_COLOR
 		return flag
+
+
+
+/datum/customer_data/japanese
+	nationality = "Space-Japanese"
+	prefix_file = "strings/names/japanese_prefix.txt"
+	base_icon = "japanese"
+	clothing_sets = list("japanese_animes")
+
+	found_seat_lines = list("Konnichiwa!", "Arigato gozaimasuuu~", "I hope there's some beef stroganoff...")
+	cant_find_seat_lines = list("I want to sit under the cherry tree already, senpai!", "Give me a seat before my Tsundere becomes Yandere!", "This place has less seating than a capsule hotel in space Tokyo!")
+	leave_mad_lines = list("I can't believe you did this! WAAAAAAAAAAAAAH!!", "I-It's not like I ever wanted your food! B-baka...")
+	leave_happy_lines = list("Oh CHEF! This is the happiest day of my life. I love you!", "I take a potato chip.... AND EAT IT!", "Itadakimasuuu~", "Gochisousama desu!")
+	wait_for_food_lines = list("No food yet? I guess it can't be helped.", "I can't wait to finally meet you burger-sama...", "Hurry up,")
+	orderable_objects = list(
+	/datum/venue/restaurant = list(/obj/item/food/tofu = 10, /datum/crafting_recipe/food/milosoup = 6, /obj/item/food/soup/vegetable = 4, /obj/item/food/sashimi = 4, /obj/item/food/chawanmushi = 4, /obj/item/food/muffin/berry = 2, /obj/item/food/beef_stroganoff = 2),
```

Comment:
let them order potato chips

Reply:
We don't have gourmet (aka non vending machine) chips, if you make some and pr it to my branch I'll add em